tools/UpdatePaths/icecube_fix.txt

```diff
@@ -0,0 +1 @@
+/turf/open/floor/grass/snow : /turf/open/floor/plating/asteroid/snow/icemoon
```

Comment:
instead of committing this file, just run it and commit the dmm changes

Reply:
Ah man, I did run it but didn't stage the changes before the commit it seems. One second fix